rules/no-array-for-each.js

```diff
@@ -315,6 +316,11 @@ function isFixable(callExpression, sourceCode, {scope, functionInfo, allIdentifi
 	return true;
 }
 
+const ignoredObjects = [
+	'React.Children',
+	'Children'
+];
```

Comment:
I was thinking we could have a shared utility method that is like `isNotNativeArrayMethod`. In case there are other false-positives in the future.

Reply:
That's a  good idea, like the [`not-dom-node.js`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/rules/utils/not-dom-node.js), but I was hoping get some help from  [`eslint-utils`](https://github.com/mysticatea/eslint-utils/issues/17), anyway I'll keep this in mind, maybe try in another PR.

Reply:
That package isn't very actively maintained. I suggest we just do the utilities here for now and we can move them there later on if it becomes maintained again. Alternatively, you could fork `eslint-utils` for now.